pkg/gatherers/conditional/parsing.go

```diff
@@ -0,0 +1,49 @@
+package conditional
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"k8s.io/klog/v2"
+)
+
+func parseGatheringRules(jsonData string) ([]GatheringRule, error) {
+	var unmarshalledRules struct {
+		Version string          `json:"version"`
```

Comment:
I think it would be good to at least log the version of the conditions IO is running with. The metadata is probably the best place where it would be useful, but let's create a new task for that. WDYT?

Reply:
It definitely will be helpful. I'll create a task for that.

Reply:
https://issues.redhat.com/browse/CCXDEV-7488